src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1422,6 +1431,14 @@ public ItemStack get(CraftItemEvent e) {
 				return e.getRecipe().getResult();
 			}
 		}, 0);
+
+		EventValues.registerEventValue(CraftItemEvent.class, Recipe.class, new Getter<Recipe, CraftItemEvent>() {
```

Comment:
move this to your event registration or the module's `#load`, same as the rest of em

Reply:
I have mixed feeling's about this as well, personally I  feel like the event-values shouldn't be separated into different files, see what others say

Reply:
I'm with efy on this one, I think it'll be fine to add `EventValues.java` to your elements package and register event values there.

Depending which direction skript goes in the future we could likely see possibility in toggling modules this will make our life easier when the time comes so we don't add if statements to every one of those.

Reply:
100% in the module